mllib/src/main/scala/org/apache/spark/ml/util/ReadWrite.scala

```diff
@@ -551,6 +551,27 @@ private[ml] object DefaultParamsWriter {
     getMetadataToSave(instance, spark, None, None)
 }
 
+
+private[ml] object MLAllowListedLoader {
+
+  def load(className: String): Class[_] = {
+    try {
+      // Only loads allow-listed classes. This is for preventing RCE in Spark Connect mode.
+      val MLHandlerClazz = Utils.classForName("org.apache.spark.sql.connect.ml.MLHandler")
```

Comment:
Should it be class/instance variables of `MLAllowListedLoader`? It is not necessary to resolve `org.apache.spark.sql.connect.ml.MLHandler` class and `safeMLClassLoader` method on every call to `load()`, right?